### ISSUE TITLE
Support for custom directives

### ISSUE THREAD

**Issue by markandrus** [fabsrc/graphql-schema-diff#17]
I have a custom directive, `@tag`. Can this tool show changes with respect to custom directives?

**a.graphql**

```graphql
directive @tag(name: String!) repeatable on FIELD_DEFINITION | INTERFACE | OBJECT | UNION | INPUT_FIELD_DEFINITION | ENUM | INPUT_OBJECT

type Foo @tag(name: "custom") {
  foo: String!
}
```

**b.graphql**

```graphql
directive @tag(name: String!) repeatable on FIELD_DEFINITION | INTERFACE | OBJECT | UNION | INPUT_FIELD_DEFINITION | ENUM | INPUT_OBJECT

type Foo {
  foo: String!
}
```

**graphql-schema-diff**

```
$ npx graphql-schema-diff a.graphql b.graphql
✔ No changes
```

**Comment by fabsrc** [comment]
Thanks for reporting this issue @markandrus! It should be fixed in [v2.5.1](https://github.com/fabsrc/graphql-schema-diff/releases/tag/v2.5.1).

**Comment by markandrus** [comment]
Thank you!!
